examples/web_apps/webxr_hand_demo/README.md

```diff
@@ -0,0 +1,65 @@
+1. At the bottom of the WebXR tab, click "Exit immersive".
+1. Observe the logged head poses in the html textarea.
+
+## VR on Quest 2 (or other headset)
+1. Using a modification of either method above, serve the content at a public URL. For example, you could upload the `task/server_files/habitat_vr_app` folder to Amazon S3 or any web host. [todo: explain how to do this for AWS]
+1. On Quest 2, open the browser and navigate to the URL. Use the Quest controller to click "Enter VR".
+1. Exit immersive VR with the Quest 2 controller's home button.
+
+# Troubleshooting
+
+If you encounter Habitat-sim build errors, make sure you have Emscripten properly installed and activated. Delete the build output folders (`habitat-sim/build_corrade-rc` and `habitat-sim/build_js`) and rebuild.
+
+If the VR app fails to load in your desktop browser, look for errors in the browser dev console. Make sure you have data files installed properly, e.g. `habitat-sim/examples/web_apps/webxr_hand_demo/data/stages/remake_v0_JustBigStuff_00.glb`. Make sure the WebXR emulator is installed and accessible (e.g. as a tab in Chrome Developer Tools).
+
+# Project folder structure
+- `data` contains all the models that are used in the default setup, plus a few extra ReplicaCAD objects
+    - `data/default.physics_config.json` contains the physics settings, e.g. gravitational acceleration
+    - `data/stages` contain the scene `.glb` files and their corresponding `.stage_config.json` files. In order to choose a scene other than `remake_v0_JustBigStuff_00`, you need to add its 2 files here.
+    - `data/objects` contains the hand models as well as the ReplicaCAD objects that can be spawned. It also contains some extra objects you may use.
+- `js` contains the JS source code of the hand demo site.
+- `lib` contains the JS transpiled version of Habitat-sim, as well as some general use utils JS files. This folder is what gets modified when you run `build_and_install_habitat_sim_js.sh`.
+- `standalone.html` is the hosted site, and `style.css` is its stylesheet.
+
+# Habitat-sim JS build
+
+The Habitat-sim JS build is a webassembly build of the Habitat simulator. It's built by compiling Habitat-sim C++ code to webassembly using the Emscripten compiler. The build outputs are `hsim_bindings.js` and `hsim_bindings.wasm`. `hsim_bindings.js` can be included from Javascript and you can call into the simulator via the bindings defined in `habitat-sim/src/esp/bindings_js/bindings_js.cpp`.
+
+The Habitat-sim JS build doesn't currently have an installer, so we can't get it from npm, conda, pip, or similar. We've added habitat-sim as a git submodule at `habitat-sim` and build it locally to produce `hsim_bindings.js` and `hsim_bindings.wasm`. We copy those files to `examples/web_apps/webxr_hand_demo/lib` as a post-build step in `build_and_install_habitat_sim_js.sh`.
```

Comment:
This text about a git submodule is incorrect for your app. Be careful about blindly copy-pasting from the `hab_vr_mephisto` demo.